src/sentry/api/serializers/models/project.py

```diff
@@ -165,6 +166,9 @@ def get_feature_list(self, obj, user):
         from sentry import features
         from sentry.features.base import ProjectFeature
 
+        if not self.include_features:
+            return None
```

Comment:
I wonder if it would be good to also kill the key in the response `serialize`

Reply:
I thought it was doing that actually. Fixed in f274062